src/State/Provider/DeserializeProvider.php

```diff
@@ -51,10 +51,7 @@ public function provide(Operation $operation, array $uriVariables = [], array $c
             return $data;
         }
 
-        if (
-            !($operation->canDeserialize() ?? true)
-            || !\in_array($method = $operation->getMethod(), ['POST', 'PUT', 'PATCH'], true)
```

Comment:
So it "_might_" be possible to deserialize a Get, GetCollection or Delete operations?
If we want to secure this behavior, maybe we should force `deserialize: false` in related operations

Reply:
Moved to MainController, see https://github.com/api-platform/core/pull/5857#discussion_r1341465506